packages/alfa-rules/test/sia-r69/rule.spec.tsx

```diff
@@ -395,6 +395,40 @@ test(`evaluate() is inapplicable to the text that is part of a label of a disabl
   t.deepEqual(await evaluate(R69, { document }), [inapplicable(R69)]);
 });
 
+test("evaluate() is inapplicable to text that would otherwise pass if it was not only punctuation", async (t) => {
+  const target = h.text(",./;'[]\\-=?<>:\"{}|_+!@#$%^&*()");
+
+  const document = h.document([
+    <html
+      style={{
+        backgroundImage: "url('foo.png')",
```

Comment:
**Suggestion:** That technically would make it a CantTell if it was applicable. `backgroundColor` would match the description better.